### PR TITLE
moved lens to use connect

### PR DESCRIPTION
**Pull request by IanAtCredo**
## Change description

Removing governance functionality from Lens and moving to it's own repo

### CODE REVIEW

File: credoai/evaluators/data_profiler.py [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 import matplotlib
 import pandas as pd
+from connect.evidence.lens_evidence import DataProfilerContainer
+
 from credoai.artifacts.data.tabular_data import TabularData
 from credoai.evaluators import Evaluator
 from credoai.evaluators.utils.validation import check_data_instance
-from credoai.evidence import ProfilerContainer
-from credoai.utils.common import ValidationError
```
Comment (esherman-credo) [inline credoai/evaluators/data_profiler.py:9]: This is just an unused import thing?

File: credoai/evaluators/ranking_fairness.py [hunk 2]
```diff
@@ -1,63 +1,183 @@
...
+            self.lb_bin = np.array(lb_bin)
+            self.ub_bin = np.array(ub_bin)
+        else:
+            self.lb_bin = lb_bin
+            self.ub_bin = ub_bin
 
     required_artifacts = ["data", "sensitive_feature"]
 
     def _setup(self):
-        self.sensitive_features = np.array(self.data.sensitive_feature)
```
Comment (esherman-credo) [inline credoai/evaluators/ranking_fairness.py]: It's not clear exactly what these changes these are for. They track back to a commit from Amin.

I can't request changes but also can't properly review.

**Review by esherman-credo — APPROVED** [review]
All comments I left were about the weird commits stacking across PRs. Looks good.

**Review by esherman-credo — COMMENTED** [review]
Tested locally with 
1. quickstart notebook (as is, without swapping to connect for governance connection)
2. governance integration notebook (as is, without swapping to connect for governance connection)
3. one of my local notebooks for testing deepchecks (as is)
4. governance integration notebook with switch to connect

Takeaways: 
- the existing Lens code (evaluators, etc.) continues to work/is not broken by this change
- connect code seems to work
- It probably makes sense to update the quickstart and governance integration docs as part of this PR to use connect.governance rather than importing Governance. If we don't, we will forget to do so until someone encounters a bug or something that doesn't work properly because they're still importing Governance the old way.

File: credoai/evaluators/data_profiler.py (hunk 1 above)
Comment (IanAtCredo) [reply credoai/evaluators/data_profiler.py:10]: Yeah, exactly.

**Comment by IanAtCredo** [comment]
I updated the code so `from credoai.governance import Governance` still works.

**Review by esherman-credo — APPROVED** [review]
Looks good

**Pull request merged**